apps/server/src/modules/account/uc/account.uc.ts

```diff
@@ -335,6 +338,12 @@ export class AccountUc {
 		}
 	}
 
+	private async checkUniqueEmail(account: AccountDto, user: User, email: string): Promise<void> {
+		if (!(await this.accountValidationService.isUniqueEmail(email, user.id, account.id, account.systemId))) {
+			throw new ValidationError(`Die E-Mail Adresse ist bereits in Verwendung!`);
```

Comment:
German message. Usually we use English for error messages.

Reply:
Done